src/nvim/lua/executor.c

```diff
@@ -103,6 +109,35 @@ static int nlua_stricmp(lua_State *const lstate) FUNC_ATTR_NONNULL_ALL
   return 1;
 }
 
+static void nlua_schedule_cb(void **argv)
+{
+  nlua_ctx *ctx = argv[0];
+  lua_State *const lstate = nlua_enter();
+  lua_rawgeti(lstate, LUA_REGISTRYINDEX, ctx->cb);
```

Comment:
We want to `nlua_unref` afterwards here, otherwise the ref table  will leak values.

Reply:
Good catch!